cfme/tests/physical_infrastructure/test_redfish_physical_server_details.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [pytest.mark.provider([RedfishProvider], scope="module")]
```

Comment:
Required:
In an integration test, we are restricting provider scope to `function` only. Please, changes scope from `module` to `function`.

Reply:
done 👍